alf/drivers/off_policy_driver.py

```diff
@@ -47,14 +47,15 @@ def __init__(self,
                 updated after every step in the environment. Each observer is a
                 callable(time_step.Trajectory).
             metrics (list[TFStepMetric]): An optiotional list of metrics.
-            training (bool): True for training, false for evaluating
+            training (bool): by default False because training info is obtained from
```

Comment:
Need to comment the meaning of `training`

Reply:
removed training flag and set it to False